pkgs/development/libraries/qt-6/modules/qtdeclarative.nix

```diff
@@ -3,16 +3,29 @@
 , qtlanguageserver
 , qtshadertools
 , openssl
+, stdenv
 , python3
+, lib
+, pkgsBuildBuild
 }:
 
 qtModule {
   pname = "qtdeclarative";
-  propagatedBuildInputs = [ qtbase qtlanguageserver qtshadertools openssl python3 ];
+  strictDeps = true;
+  propagatedBuildInputs = [ qtbase qtlanguageserver qtshadertools openssl ];
+  nativeBuildInputs = [ python3 ];
   patches = [
     # prevent headaches from stale qmlcache data
     ../patches/qtdeclarative-default-disable-qmlcache.patch
     # add version specific QML import path
     ../patches/qtdeclarative-qml-paths.patch
   ];
+  # Conditional is required to prevent infinate recursion during a cross build
```

Comment:
infinate -> infinite

Reply:
is this comment still needed? it's not above conditional, maybe you removed the conditional?